changelog.txt

```diff
@@ -38,9 +38,10 @@ Template for new versions:
 - `position`: support for adv mode look cursor
 
 ## Misc Improvements
-- `hide-tutorials`: handle tutorial popups for adventure mode
-- `hide-tutorials`: new ``reset`` command that will re-enable popups in the current game (in case you hid them all and now want them back)
-- `gui/notify`: moody dwarf notification turns red when they can't reach workshop or items
+- `hide-tutorials`: if enabled, also hide tutorial popups for adventure mode
+- `hide-tutorials`: new ``reset`` command that will re-enable popups in the current game
+- `gui/notify`: save reminder now appears in adventure mode and also changes color to yellow at 30 minutes and to orange at 60 minutes
+- `gui/notify`: moody dwarf notification turns red when they can't find workshop or items
```

Comment:
```suggestion
- `hide-tutorials`: handle tutorial popups for adventure mode
- `hide-tutorials`: new ``reset`` command that will re-enable popups in the current game (in case you hid them all and now want them back)
- `gui/notify`: moody dwarf notification turns red when they can't reach workshop or items
- `gui/notify`: save reminder now appears in adventure mode
- `gui/notify`: save reminder changes color to yellow at 30 minutes and to orange at 60 minutes
```